src/CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ if( NOT XRDCL_ONLY )
 
   add_subdirectory( XrdMacaroons )
   add_subdirectory( XrdVoms )
+  add_subdirectory( XrdCors )
```

Comment:
We are trying to group anything specific to http in directories that start with XrdHttp (notice that XrdTpc was changed to XrdHttpTpc). So, it would follow that this should be named XrdHttpCors.